pkgs/development/libraries/liberfa/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "erfa";
+  version = "1.7.0";
+
+  buildInputs = [ autoreconfHook ];
+
+  src = fetchFromGitHub {
+    owner = "liberfa";
+    repo = "erfa";
+    rev = "v${version}";
+    sha256 = "1z4k2phrw6wwi0kax6ac80jk9c036gi7pmhmg6gaf3lk81k6xz2r";
+  };
+
+  preConfigure = "./bootstrap.sh";
```

Comment:
Have you tried discarding it? Our `autoreconfHook` usually runs the standard commands most source put in `./bootstrap.sh`.

Reply:
Yes, this is not needed. I remove it.